src/lib/pickles/intf.mli

```diff
@@ -231,24 +149,37 @@ module type Inputs_base = sig
 
     val size : Import.B.t
 
+    (** The size in bits for the canonical representation of a field
+        element *)
     val size_in_bits : int
 
+    (** [to_bits x] returns the little endian representation of the canonical
+        representation of the field element [x] *)
     val to_bits : t -> bool list
 
+    (** [of_bits bs] builds an element of the field using the little endian
+        representation given by [bs] *)
     val of_bits : bool list -> t
 
+    (** [is_square y] returns [true] if there exists an element [x] in the same
+        field such that [x^2 = y] *)
```

Comment:
Maybe also note `y` is a  quadratic residue modulo `p`

Reply:
Updated.